package.json

```diff
@@ -32,14 +32,14 @@
     "plugins": "npm run plugins:install && npm run plugins:prepare",
     "plugins:install": "node scripts/installExternalPlugins.js",
     "plugins:prepare": "node scripts/prepareInstalledPlugins.js",
-    "test": "NODE_ENV=test ava --c $(node -p 'Math.max(os.cpus().length - 2, 1)')",
-    "test:server": "cp -n .env.example .env && yarn && NODE_ENV=test ava --c $(node -p 'Math.max(os.cpus().length - 2, 1)')",
```

Comment:
`cp -n .env.example .env` is not only there for users who may have forgotten to create their `.env` file, but as a default env for plugins too when running server tests after self-loading.